### DISCUSSION TITLE
[Icon Request] Food+drink icon

### DISCUSSION THREAD

**Discussion by Malthehave** [tailwindlabs/heroicons#465] [Ideas]
Hey,

I thought it would be cool to add a food and or drink icon. Sort of like [this](https://icons-for-free.com/iconfiles/png/512/coke+coke+and+burger+drink+fast+food+icon-1320167994567745412.png)

You could also call it a restaurant or fast food icon.

Thanks for a great set of icons!

**Comment by aspearman** [comment]
Fork and knife, plate, fork and knife on top of plate, takeout bag would all be great.
